kernel/nativecode.ml

```diff
@@ -1280,9 +1280,6 @@ let ml_of_instance instance u =
   | Lmakeblock (prefix,(cn,_u),_,args) ->
      let args = Array.map (ml_of_lam env l) args in
      MLconstruct(prefix,cn,args)
-  | Lconstruct (prefix, (cn,u)) ->
-     let uargs = ml_of_instance env.env_univ u in
-      mkMLapp (MLglobal (Gconstruct (prefix, cn))) uargs
```

Comment:
It looks like this is the only place where we generate a `Gconstruct` node, which means it is now dead code. You should probably remove it as well.

Reply:
Oh, good point, let me do it as part of this PR, since it will simplify some further patches.

Reply:
Done.